packages/cli/src/commands/setup/tailwind/tailwind.js

```diff
@@ -157,7 +164,7 @@ export const handler = async ({ force }) => {
            * Later, when we can tell the vscode extension where to look for the config,
            * we can put it in web/config/
```

Comment:
I actually don't think this comment is true anymore. I think it must live in the root of "web" in order to be picked up by the extension.

Reply:
Exactly, what's why we have it there for the time being. In the future, if it becomes possible to tell the extension where to look, we can move it into web/config/

Reply:
Aaaaah-ha!